wikitemplate.md

```diff
@@ -170,11 +170,14 @@
 - [ ] Turning Safe Browsing off and shields off both disable safe browsing for https://www.raisegame.com/.
 - [ ] Visit https://brianbondy.com/ and then turn on script blocking, nothing should load. Allow it from the script blocking UI in the URL bar and it should work.
 - [ ] Test that about:preferences default Bravery settings take effect on pages with no site settings.
-- [ ] Test that turning on fingerprinting protection in about:preferences shows 3 fingerprints blocked at https://jsfiddle.net/bkf50r8v/13/. Test that turning it off in the Bravery menu shows 0 fingerprints blocked.
 - [ ] Test that 3rd party storage results are blank at https://jsfiddle.net/7ke9r14a/9/ when 3rd party cookies are blocked and not blank when 3rd party cookies are unblocked.
-- [ ] Test that audio fingerprint is blocked at https://audiofingerprint.openwpm.com/ when fingerprinting protection is on.
+
+**Fingerprinting Tests (Enable fingerprinting protection in `about:preferences#shields`)**
+- [ ] Visit https://jsfiddle.net/bkf50r8v/13/, ensure 3 blocked items are listed in shields. Result window should show `got canvas fingerprint 0`  and  `got webgl fingerprint 00`
+- [ ] Visit https://audiofingerprint.openwpm.com/, ensure 2 blocked items are listed in shields. Audio fingerprint graph is shown but doens't contain user fingerprint graph represented in blue line
 - [ ] Test that browser is not detected on https://extensions.inrialpes.fr/brave/
```

Comment:
This should be moved above the `Fingerprinting Tests` as it's not related to FP... Test ensures that we're appearing as Chrome.